src/app/shared/post-capture-card/post-capture-card.component.ts

```diff
@@ -11,10 +10,11 @@ export class PostCaptureCardComponent implements OnInit {
 
   @Input() transaction!: Transaction;
   @Input() asset!: Asset;
+  @ViewChild('igIMG')
```

Comment:
`igIMG` is a bad naming. First, it consists of two __abbreviation__: `ig` and `IMG`. This will greatly decrease the readability. Second, it should follow camelCase.